custom_tracing/Cargo.toml

```diff
@@ -6,22 +6,23 @@ license.workspace = true
 edition.workspace = true
 
 [dependencies]
-serde = { version = "1", features = ["derive"] }
-notify = { version = "6", default-features = false }
 anyhow = "1"
+axum = { version = "0.7", features = ["tracing"], optional = true }
+bs58 = { version = "0.4", features = ["check"] }
```

Comment:
don'tt be this precise in a lib this far upstream? Make version = "0" or version = ">=..." if it needs a minimum version